### PR TITLE
Disables template and workflow template fields.

### PR DESCRIPTION
**Pull request by AlexSCorey**
##### SUMMARY
This addresses #7973. It also addresses a bug where a user might temporarily not have labels while the label select component is doing its fetching and loading.  I did file a feature request with PF that would allow use to disable all child of `<InputGroup>` so that we would not have add styling to `<ChipHolder>` component in the template form. https://github.com/patternfly/patternfly-react/issues/4709

##### ISSUE TYPE
 - Bugfix Pull Request

##### COMPONENT NAME
 - UI

##### ADDITIONAL INFORMATION
Screen Shot 2020-08-24 at 12 11 17 PM
Screen Shot 2020-08-24 at 12 11 02 PM

### CODE REVIEW

File: awx/ui_next/src/components/Lookup/Lookup.jsx [hunk 1]
```diff
@@ -103,11 +106,15 @@ function Lookup(props) {
...
           variant={ButtonVariant.control}
-          isDisabled={isLoading}
+          isDisabled={isLoading || isDisabled}
         >
           <SearchIcon />
         </Button>
-        <ChipHolder className="pf-c-form-control">
+        <ChipHolder
+          isDisabled={isDisabled}
+          // css="background-color: #d2d2d2"
```
Comment (jlmitch5) [inline awx/ui_next/src/components/Lookup/Lookup.jsx]: can you remove this comment

**Review by jlmitch5 — APPROVED** [review]
looks good, just a super small comment removal change

**Comment by unlikelyzero** [comment]
Notes: this would be a great network server intercept to verify that no additional API calls are made that would ultimately end in a 403

**Review by tiagodread — CHANGES_REQUESTED** [review]
In the legacy UI, the fields: Inventory (and prompt on launch), project, playbook, Webhook service are disabled for Job templates.

image
 
In the UI Next, Prompt on launch and Webhook service are enabled for edit, but when I try to save it I don’t have permission.

limited_user

**Comment by tiagodread** [comment]
You can also enable "Prompt on launch" for WFJT, but when you save, an error message is displayed:

image

**Comment by tiagodread** [comment]
- Trying to select the options `Privilege Escalation Provisioning Callbacks Enable Webhook Concurrent Jobs Enable Fact Storage` the error message "You do not have permission to perform this action." is displayed:

image
 @AlexSCorey should this inputs be enabled in this case?

**Comment by AlexSCorey** [comment]
recheck

**Comment by AlexSCorey** [comment]
recheck

**Review by tiagodread — APPROVED** [review]
Everything ok

**Comment by AlexSCorey** [comment]
regate

**Comment by AlexSCorey** [comment]
recheck

**Pull request merged**